### ISSUE TITLE
 different headerbar color when window is active/unactive

### ISSUE THREAD

**Issue by demostanis** [themix-project/oomox-gtk-theme#90]
moved (again) from https://github.com/themix-project/themix-plugin-base16/issues/6
hope this is the right place now...

is there an option/would it be possible to have an option to indicate a different color for the headerbar when its window is active and unactive?
i tried to change WM_BORDER_{UN,}FOCUS but that did not have any effect:
i have a [script](https://github.com/demostanis/demolinux2/blob/master/airootfs/opt/oomox/run_oomox.sh) which does the same as oomox-gui: using the base16 plugin to generate colors which i pipe to change_color.sh. i have a [file](https://github.com/demostanis/demolinux2/blob/master/airootfs/opt/oomox/additions) which contains additional variables.
i set WM_BORDER_FOCUS (i didn't commit it) to 000000 but i see no change in the headerbar' colors

**Comment by actionless** [comment]
isn't it the same thing as here? https://github.com/themix-project/oomox-gtk-theme/issues/18

**Comment by demostanis** [comment]
i see that i can monkey patch the scss files to get the effect i want, but shouldnt there be a variable to do that?

**Comment by actionless** [comment]
yup, you can add one, but it should be then in sync with all the parts of the theme, not just for one toolkit

**Comment by demostanis** [comment]
> i see that i can monkey patch the scss files to get the effect i want, but shouldnt there be a variable to do that?
actually, no, it seems that setting titlebar_bg_unfocused doesnt do anything, the titlebar's color when unfocused stays the same when focused (however, changing titlebar_bg_focused does work)
